data/pan-cov-example/peptide_table.csv

```diff
@@ -1,4 +1,4 @@
-peptide_id,Virus,Strain,Protein,Full_name,Oligo,Prot,Prot_Start
+peptide_id,Virus,Strain,Protein,Full_name,oligo,Prot,Prot_Start

```

Comment:
I'm sure you've noticed this, but the code seems to expect `Oligo` in many places

Reply:
yup, trying to use the lowercase version - cleaning up now.